static/admin/config.yml

```diff
@@ -284,3 +383,84 @@ collections:
                   hint: "The person / organization who said the quote",
                 }
               - { label: Quote, name: quote, widget: string }
+
+      - label: "Recruitment Page"
+        name: "recruitment"
+        file: "/src/markdown/joinUs.md"
+        fields:
+          - { label: Heading, name: heading, widget: string }
+          - {
+              label: SEO Page Description,
+              name: meta_page_description,
+              widget: text,
+              required: false,
+              hint: "This text will show up in the Google description of the page (The text under the link)",
+            }
+          - label: Recruitment Categories
+            name: recruitment_categories
+            widget: list
+            fields:
+              - { label: Name, name: name, widget: string }
+              - { label: Currenrtly Open, name: is_open, widget: boolean }
+              - {
+                  label: Description (Open),
+                  name: description,
+                  widget: text,
+                  hint: Description when recruitments are open.,
```

Comment:
I think u need quotations around the text here?

Reply:
I don't think it's really needed since it's automatically read as a string. It's only needed for empty values and escaping reserved words like `true` and `false`

[YAML is weird](https://hitchdev.com/strictyaml/why/implicit-typing-removed/), I know